sources/packages/backend/workflow/test/test-utils/constants/system-configurations-constants.ts

```diff
@@ -1,5 +1,16 @@
+import { Duration } from "zeebe-node";
+
 /**
  * Timeout in milliseconds for the zeebe client to create process instance
  * and wait for response.
  */
 export const PROCESS_INSTANCE_CREATE_TIMEOUT = 90000;
+/**
+ * While publishing a message to the workflow, this is a maximum amount of time
+ * to wait before the message is discarded.
+ * The E2E tests can potentially send a message to the workflow instants before the
+ * workflow is even waiting for the message to arrive. This time ensures that the
+ * message will continue to wait for few seconds, once it is received by Camunda,
+ * and not be dropped immediately.
+ */
+export const PUBLISH_MESSAGE_TIME_TO_LEAVE_SECONDS = Duration.seconds.of(10);
```

Comment:
If we assume that e2e tests and workers are approximately going to have same value, then we can the variable for shared lib.

`export const ZEEBE_PUBLISH_MESSAGE_DEFAULT_TIME_TO_LEAVE = Duration.seconds.of(10);`

Reply:
We can but I avoid assuming that they can be different.